plugins/remmina/__init__.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+"""Search and start Remmina connections."""
+
+from albertv0 import *
+import os
+import subprocess
+from glob import glob
+from re import search, IGNORECASE
+import configparser
+
+__iid__ = "PythonInterface/v0.2"
+__prettyname__ = "Remmina"
+__version__ = "0.1"
+__trigger__ = "rm"
+__author__ = "Oğuzcan Küçükbayrak"
+__dependencies__ = ["remmina"]
+
+iconPath = "/usr/share/icons/hicolor/scalable/apps/org.remmina.Remmina.svg"
+connectionsPath = "%s/.local/share/remmina" % (os.environ['HOME'])
+protoIconPath = "/usr/share/icons/hicolor/scalable/emblems/remmina-%s-symbolic.svg"
```

Comment:
what's the difference between this and the `iconPath`?

Reply:
`protoIconPath` is the icon of the connection's protocol. For example; `remmina-rdp-symbolic.svg` or `remmina-ssh-symbolic.svg`. I'll check the sizes and try to pack some of the icons with the module if they are not too large as requested with the above comment.